### PR TITLE
feat(upgrade): rectify image tag

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

**Review by w3aman — APPROVED** [review]
/lgtm

**Review by niladrih — APPROVED** [review]

**Comment by sinhaashish** [comment]
bors merge

File: k8s/upgrade/src/constant.rs [hunk 1]
```diff
@@ -62,9 +62,9 @@ pub(crate) fn upgrade_event_selector(release_name: &str, component_name: &str) -
 }
 
 /// Upgrade conatainers to develop.
```
Comment (avishnu) [inline k8s/upgrade/src/constant.rs:64]: Typo in containers

File: k8s/upgrade/src/constant.rs [hunk 2]
```diff
@@ -62,9 +62,9 @@ pub(crate) fn upgrade_event_selector(release_name: &str, component_name: &str) -
...
-pub(crate) const UPGRADE_JOB_TO_DEVELOP_TAG: &str = "jai-shree-ram";
+pub(crate) const UPGRADE_JOB_TO_DEVELOP_TAG: &str = "develop";
 /// Upgrade job container image repository.
-pub(crate) const UPGRADE_JOB_IMAGE_REPO: &str = "sinhaashish";
+pub(crate) const UPGRADE_JOB_IMAGE_REPO: &str = "openebs";
 /// Upgrade job container image name.
 pub(crate) const UPGRADE_JOB_IMAGE_NAME: &str = "mayastor-upgrade-job";
 /// Upgrade job name suffix.
```
Comment (avishnu) [inline k8s/upgrade/src/constant.rs:70]: How are the names hard-coded here?

Comment (sinhaashish) [reply k8s/upgrade/src/constant.rs:70]: only these two

**Pull request merged**